src/App/Services/Structure.php

```diff
@@ -2,7 +2,7 @@
 
 namespace LaravelEnso\Upgrade\App\Services;
 
-use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Support\Collection;
```

Comment:
Note: `Permission::whereIn('name', $permissions)->pluck('name');` at line 30 returns an instance of `Illuminate\Support\Collection`